src/foam/i18n/LocaleTranslationService.js

```diff
@@ -20,8 +29,24 @@ foam.CLASS({
     {
       name: 'getTranslation',
       javaCode: `
-      System.err.println("*********************************** getTranslation");
-      return "foobar";
+        if ( locale.equals("en") ) return originalText;
```

Comment:
Always translate.  Let the cache take care of this.

Reply:
Yes, it is surprisingly common for people to translate english text.